src/client/containers/authentication/authentication.js

```diff
@@ -43,7 +39,7 @@ class Authentication extends Component {
         path = '/login';
         data = { email, password };
       }
-      post(path, data, history);
+      this.props.onAuth(path, data, history);
```

Comment:
Why don't you use destruction for onAuth?

Reply:
I prefer to not destructing props, but anyway changed